packages/cozy-client/src/CozyClient.js

```diff
@@ -1,5 +1,5 @@
 import StackLink from './StackLink'
-import Association from './associations'
+import createAssociation from './associations'
```

Comment:
Minor: 

I would not `export default createAssociation` inside './associations'.

I would `export { create }` and then here `import { create as createAssociation } from './associations'`

Reply:
:+1: